src/org/zaproxy/zap/extension/fuzz/httpfuzzer/processors/tagcreator/RegExTagRule.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2015 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.fuzz.httpfuzzer.processors.tagcreator;
+
+public abstract class RegExTagRule implements ITagRule {
+    private String _regExp;
+
+
+    protected RegExTagRule(String regExp) {
+        _regExp = regExp;
+    }
+
+    public String getRegExp(){
+        return _regExp;
+    }
+
+    public abstract String createTag(String responseMessage);
```

Comment:
The declaration of the method should be removed (it's redundant, it's already defined in interface).